stable/jenkins/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 name: jenkins
 home: https://jenkins.io/
-version: 1.21.2
+version: 1.21.4
```

Comment:
1.22.0 would be ok as it's more than a patch ;-)

Reply:
agree...not sure what I was thinking :-)